openquake/mbt/tests/tools/fault_modeler/test_fault_source_modeler.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+
+import openquake.mbt.tools.fault_modeler.fault_source_modeler as fsm
+
+# -----------------------------------------------------------------------------
+
+param_map = {'source_id': 'ogc_fid',
```

Comment:
Have global variables isn't the best choice here, you have, at least, 3 alternatives:
1) continue to use them in this way but stop to prefix methods variable with underscore to distinguish them from globals
1) add a class like ``Globals`` that return these values from classmethods (so you can avoid to use deepcopy)
2) convert them to immutables constants and get them properly (with a cast)

Reply:
I hope I managed to fix everything. @cossatot please double check